src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataEntityReferenceLinkWrapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// Encapsulates an <see cref="ODataEntityReferenceLink"/> .
+    /// </summary>
+    public sealed class ODataEntityReferenceLinkWrapper : ODataItemBase
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="ODataEntityReferenceLinkWrapper"/>.
+        /// </summary>
+        /// <param name="item">The wrapped item.</param>
+        public ODataEntityReferenceLinkWrapper(ODataEntityReferenceLink item)
+            : base(item)
+        {
+        }
+
+        /// <summary>
+        /// Gets the wrapped <see cref="ODataEntityReferenceLink"/>.
+        /// </summary>
+        public ODataEntityReferenceLink EntityReferenceLink
+        {
+            get
+            {
+                return Item as ODataEntityReferenceLink;
+            }
+        }
```

Comment:
change it as 

public ODataEntityReferenceLink EntityReferenceLink {get;}

in the ctor, set this value as:

EntityReferenceLink  = item;

Reply:
resolved